### ISSUE TITLE
Invalid user name or password

### ISSUE THREAD

**Issue by sdf611097** [goharbor/harbor-helm#75]
After helm install XXX, and wait for all pods running.
I enter the portal using default username/password ( admin/Harbor12345), I got "Invalid user name or password".

I have no idea why I cannot login.

Note: I had checked Harbor12345 is defined in secret

**Comment by ywk253100** [comment]
Check the logs of Harbor core.

**Comment by ywk253100** [comment]
@aaronnew2018 Are you using `NodePort`? Can you try to curl the homepage and check the output?

**Comment by aaronnew2018** [comment]
@ywk253100 yes, I was using NodePort for Harbor-Portal, and the curl as below:
<!doctype html>
<html>

<head>
    <meta charset="utf-8">
    <title>Harbor</title>
    <base href="/">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="icon" type="image/x-icon" href="favicon.ico?v=2">
<link rel="stylesheet" href="styles.0a97e5c7dae15939bfca.css"></head>

<body>
    <harbor-app>
        <div class="spinner spinner-lg app-loading">
            Loading...
        </div>
    </harbor-app>
<script type="text/javascript" src="runtime.a66f828dca56eeb90e02.js"></script><script type="text/javascript" src="scripts.70b8b612aef2adc60282.js"></script><script type="text/javascript" src="main.7fa8cf5667e763a02956.js"></script></body>

**Comment by ywk253100** [comment]
What is the version of Harbor chart that you are using? 
Can you try to debug what happens? You can open the debug tool of browser and check the API call sent to backend and check the logs of Harbor.

**Comment by aaronnew2018** [comment]
@ywk253100 the dev version in harbor-helm master branch.   The same logs as above bro. : 

Request URL: https://repository.rhinopithecus.club/c/login
Request Method: POST
Status Code: 405 
Remote Address: 185.188.6.62:443
Referrer Policy: no-referrer-when-downgrade
content-length: 575
content-type: text/html
date: Thu, 08 Nov 2018 03:48:23 GMT
server: nginx/1.14.0 (Ubuntu)
status: 405
:authority: repository.rhinopithecus.club
:method: POST
:path: /c/login
:scheme: https
accept: application/json, text/plain, */*
accept-encoding: gzip, deflate, br
accept-language: en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7
content-length: 35
content-type: application/x-www-form-urlencoded
cookie: ajs_group_id=null; ajs_user_id=%22j69hcn8og3feznrhn3o8arocyw%22; ajs_anonymous_id=%2200000000000000000000000000%22; harbor-lang=en-us
origin: https://repository.rhinopithecus.club
referer: https://repository.rhinopithecus.club/harbor/sign-in
user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36

**Comment by ywk253100** [comment]
Can you show me the logs of `nginx`, `portal` and `core` when you posting `/c/login`?

**Comment by stevesloka** [comment]
I'm seeing the same issues where after deployment, I cannot log in. I've included the logs below but they don't offer much. 

I've also attempted the `0.3.0` branch, but it has an error about missing the `redis` chart. Additionally, the images the helm chart uses differs from the ones in the main repo if you do an offline/online install. Is there a reason for the difference? (This is moving off of this thread, so happy to open a new issue so I don't muddy this one). 

Core logs:
```
2018-11-13T21:20:54Z [DEBUG] [authenticator.go:126]: Registered authencation helper for auth mode: db_auth
2018-11-13T21:20:54Z [DEBUG] [authenticator.go:126]: Registered authencation helper for auth mode: ldap_auth
2018-11-13T21:20:54Z [DEBUG] [authenticator.go:126]: Registered authencation helper for auth mode: uaa_auth
2018-11-13T21:20:54Z [DEBUG] [init.go:37]: topic StartReplication is subscribed
2018-11-13T21:20:54Z [DEBUG] [init.go:37]: topic OnPush is subscribed
2018-11-13T21:20:54Z [DEBUG] [init.go:37]: topic OnDeletion is subscribed
2018-11-13T21:20:54Z [INFO] Config path: /etc/core/app.conf
2018-11-13T21:20:54Z [INFO] initializing configurations...
2018-11-13T21:20:54Z [INFO] key path: /etc/core/key
2018-11-13T21:20:54Z [INFO] initializing client for adminserver http://harbor-harbor-adminserver ...
2018-11-13T21:20:54Z [ERROR] [utils.go:101]: failed to connect to tcp://harbor-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.111.159.38:80: getsockopt: connection refused
2018-11-13T21:20:58Z [ERROR] [utils.go:101]: failed to connect to tcp://harbor-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.111.159.38:80: i/o timeout
2018-11-13T21:21:00Z [INFO] initializing the project manager based on local database...
2018-11-13T21:21:00Z [INFO] configurations initialization completed
2018-11-13T21:21:00Z [INFO] Registering database: type-PostgreSQL host-harbor-harbor-database port-5432 databse-registry sslmode-"disable"
2018-11-13T21:21:00Z [INFO] Register database completed
2018-11-13T21:21:00Z [INFO] User id: 1 already has its encrypted password.
2018-11-13T21:21:00Z [INFO] Enable redis cache for chart caching
2018-11-13T21:21:00Z [DEBUG] [chart_repository.go:503]: Chart storage server is set to http://harbor-harbor-chartmuseum
2018-11-13T21:21:00Z [INFO] API controller for chart repository server is successfully initialized
2018-11-13T21:21:00Z [INFO] initialized clair database
2018-11-13T21:21:00Z [INFO] Because SYNC_REGISTRY set false , no need to sync registry 
2018-11-13T21:21:00Z [INFO] Init proxy
2018/11/13 21:21:00 [I] [asm_amd64.s:2337] http server Running on http://:8080
```

Portal logs:
```
192.168.109.129 - - [13/Nov/2018:21:25:50 +0000] "GET /api/users/current HTTP/1.1" 200 693 "https://harbor.containersteve.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"
192.168.109.129 - - [13/Nov/2018:21:25:50 +0000] "GET /api/systeminfo HTTP/1.1" 200 693 "https://harbor.containersteve.com/harbor/sign-in" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"``

**Comment by ywk253100** [comment]
@stevesloka Seems the request is not received by Harbor Core, can you check the debug console of browser and check that the request is sent out. And check the logs of ingress controller or nginx(if you use NodePort).

For the chart on 0.3.0 branch, you must run `helm dependency update` first before installing it as said in the guide https://github.com/goharbor/harbor-helm/tree/0.3.0.

The images used by chart of 0.3.0 branch are Harbor 1.6.0, they are same with the online/offline package of version 1.6.0. And the chart of master branch is using the dev images. They are changed by code merging.

**Comment by albamc** [comment]
I have same problems (invalid username or password)
And I found these browser's (chrome) console logs.

```
ERROR Error: Uncaught (in promise): SyntaxError: Unexpected token < in JSON at position 0
SyntaxError: Unexpected token < in JSON at position 0
...
Failed to load bootstrap options with error:  SyntaxError: Unexpected token < in JSON at position 0
...
```

**Comment by ywk253100** [comment]
@albamc How about the requests sent to backend? You can find them in the `Network` tab:
![qq20181121-095752](https://user-images.githubusercontent.com/5835782/48813991-1a6adb80-ed74-11e8-87be-6f741ae5baa9.png)

**Comment by andreiapostol22** [comment]
same issue here as well.

**Comment by silentdark** [comment]
same issue

core logs:

```
2018-11-26T10:02:54Z [DEBUG] [authenticator.go:126]: Registered authencation helper for auth mode: db_auth
2018-11-26T10:02:54Z [DEBUG] [authenticator.go:126]: Registered authencation helper for auth mode: ldap_auth
2018-11-26T10:02:54Z [DEBUG] [authenticator.go:126]: Registered authencation helper for auth mode: uaa_auth
2018-11-26T10:02:54Z [DEBUG] [init.go:37]: topic StartReplication is subscribed
2018-11-26T10:02:54Z [DEBUG] [init.go:37]: topic OnPush is subscribed
2018-11-26T10:02:54Z [DEBUG] [init.go:37]: topic OnDeletion is subscribed
2018-11-26T10:02:54Z [INFO] Config path: /etc/core/app.conf
2018-11-26T10:02:54Z [INFO] initializing configurations...
2018-11-26T10:02:54Z [INFO] key path: /etc/core/key
2018-11-26T10:02:54Z [INFO] initializing client for adminserver http://hub-harbor-adminserver ...
2018-11-26T10:02:55Z [ERROR] [utils.go:101]: failed to connect to tcp://hub-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.103.110.178:80: i/o timeout
2018-11-26T10:02:59Z [ERROR] [utils.go:101]: failed to connect to tcp://hub-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.103.110.178:80: i/o timeout
2018-11-26T10:03:05Z [ERROR] [utils.go:101]: failed to connect to tcp://hub-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.103.110.178:80: i/o timeout
2018-11-26T10:03:15Z [ERROR] [utils.go:101]: failed to connect to tcp://hub-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.103.110.178:80: i/o timeout
2018-11-26T10:03:33Z [ERROR] [utils.go:101]: failed to connect to tcp://hub-harbor-adminserver:80, retry after 2 seconds :dial tcp 10.103.110.178:80: i/o timeout
2018-11-26T10:03:35Z [INFO] initializing the project manager based on local database...
2018-11-26T10:03:35Z [INFO] configurations initialization completed
2018-11-26T10:03:35Z [INFO] Registering database: type-PostgreSQL host-hub-harbor-database port-5432 databse-registry sslmode-"disable"
2018-11-26T10:03:35Z [INFO] Register database completed
2018-11-26T10:03:35Z [INFO] User id: 1 updated its encypted password successfully.
2018-11-26T10:03:35Z [INFO] Enable redis cache for chart caching
2018-11-26T10:03:35Z [DEBUG] [chart_repository.go:503]: Chart storage server is set to http://hub-harbor-chartmuseum
2018-11-26T10:03:35Z [INFO] API controller for chart repository server is successfully initialized
2018-11-26T10:03:36Z [INFO] initialized clair database
2018-11-26T10:03:36Z [INFO] Because SYNC_REGISTRY set false , no need to sync registry 
2018-11-26T10:03:36Z [INFO] Init proxy
2018/11/26 10:03:36 [1;34m[I] [asm_amd64.s:2337] http server Running on http://:8080[0m
2018-11-26T10:03:41Z [DEBUG] [security.go:233]: can not get user information from session
2018-11-26T10:03:41Z [DEBUG] [security.go:290]: user information is nil
2018-11-26T10:03:41Z [DEBUG] [security.go:304]: using local database project manager
2018-11-26T10:03:41Z [DEBUG] [security.go:306]: creating local database security context...
2018-11-26T10:03:41Z [DEBUG] [security.go:233]: can not get user information from session
2018-11-26T10:03:41Z [DEBUG] [security.go:290]: user information is nil
2018-11-26T10:03:41Z [DEBUG] [security.go:304]: using local database project manager
2018-11-26T10:03:41Z [DEBUG] [security.go:306]: creating local database security context...
 ```

**Comment by ywk253100** [comment]
@andreiapostol22 @silentdark 
1. Make sure all pods are in running status, if not, check the logs.
2. Provide the console log of the browser as described in https://github.com/goharbor/harbor-helm/issues/75#issuecomment-440500292
3. How do you expose the service? Ingress or NodePort?
4. Provide the logs of ingress controller(if using ingress) or Nginx(if using NodePort)

**Comment by ywk253100** [comment]
Closing the issue as doesn't get any feedback.

**Comment by nick-lehmann** [comment]
I would like to see this issue reopened as I am facing the same problem. @ywk253100 see below for the information that you have requested.

# Environment

Used version of Kubernetes

```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.11", GitCommit:"637c7e288581ee40ab4ca210618a89a555b6e7e9", GitTreeState:"clean", BuildDate:"2018-11-26T14:38:32Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:02:58Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

I have installed all components of Harbor into their own namespace (`harbor`). Here is the output of `kubectl -n harbor get all`. As far as I was concerned, this looked promising.

```
NAME                                             READY     STATUS    RESTARTS   AGE
pod/harbor-harbor-chartmuseum-6f6cd9f5b5-dmp2n   1/1       Running   0          19m
pod/harbor-harbor-clair-5949867d67-p9nnw         1/1       Running   5          19m
pod/harbor-harbor-core-c6566678d-k79h5           1/1       Running   2          19m
pod/harbor-harbor-database-0                     1/1       Running   0          19m
pod/harbor-harbor-jobservice-59bb7fd5db-zgv6d    1/1       Running   2          19m
pod/harbor-harbor-portal-849fdb497d-kkmxc        1/1       Running   0          19m
pod/harbor-harbor-redis-0                        1/1       Running   0          19m
pod/harbor-harbor-registry-78f6f945b7-c8sb7      2/2       Running   0          19m

NAME                                                             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
service/glusterfs-dynamic-f09751b2-6910-11e9-a684-0050563e2073   ClusterIP   10.104.53.69     <none>        1/TCP               20m
service/harbor-harbor-chartmuseum                                ClusterIP   10.102.104.185   <none>        80/TCP              19m
service/harbor-harbor-clair                                      ClusterIP   10.106.95.161    <none>        6060/TCP,6061/TCP   19m
service/harbor-harbor-core                                       ClusterIP   10.100.12.123    <none>        80/TCP              19m
service/harbor-harbor-database                                   ClusterIP   10.99.236.170    <none>        5432/TCP            19m
service/harbor-harbor-jobservice                                 ClusterIP   10.96.45.91      <none>        80/TCP              19m
service/harbor-harbor-portal                                     ClusterIP   10.110.23.228    <none>        80/TCP              19m
service/harbor-harbor-redis                                      ClusterIP   10.107.47.233    <none>        6379/TCP            19m
service/harbor-harbor-registry                                   ClusterIP   10.99.129.132    <none>        5000/TCP,8080/TCP   19m

NAME                                        DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/harbor-harbor-chartmuseum   1         1         1            1           19m
deployment.apps/harbor-harbor-clair         1         1         1            1           19m
deployment.apps/harbor-harbor-core          1         1         1            1           19m
deployment.apps/harbor-harbor-jobservice    1         1         1            1           19m
deployment.apps/harbor-harbor-portal        1         1         1            1           19m
deployment.apps/harbor-harbor-registry      1         1         1            1           19m

NAME                                                   DESIRED   CURRENT   READY     AGE
replicaset.apps/harbor-harbor-chartmuseum-6f6cd9f5b5   1         1         1         19m
replicaset.apps/harbor-harbor-clair-5949867d67         1         1         1         19m
replicaset.apps/harbor-harbor-core-c6566678d           1         1         1         19m
replicaset.apps/harbor-harbor-jobservice-59bb7fd5db    1         1         1         19m
replicaset.apps/harbor-harbor-portal-849fdb497d        1         1         1         19m
replicaset.apps/harbor-harbor-registry-78f6f945b7      1         1         1         19m

NAME                                      DESIRED   CURRENT   AGE
statefulset.apps/harbor-harbor-database   1         1         19m
statefulset.apps/harbor-harbor-redis      1         1         19m
```

Harbor is exposed by my `traefik` ingress controller. The following file is the `values.yaml` file that I have passed to the `helm install` command.

```
expose:
  type: ingress
  tls:
    enabled: false
    secretName: ""
    commonName: ""

  ingress:
    host: hub.nicklehmann.me
    controller: default
    annotations:
      kubernetes.io/ingress.class: traefik
      traefik.ingress.kubernetes.io/rule-type: "PathPrefixStrip"
    rewriteAnnotation: nginx.ingress.kubernetes.io/rewrite-target

externalURL: https://hub.nicklehmann.me

persistence:
  enabled: true
  resourcePolicy: "keep"
  persistentVolumeClaim:

    registry:
      existingClaim: "harbor"
      storageClass: "gluster-heketi"
      subPath: "registry"
      accessMode: ReadWriteOnce
      size: 40Gi

    chartmuseum:
      existingClaim: "harbor"
      storageClass: ""
      subPath: "chartmuseum"
      accessMode: ReadWriteOnce
      size: 10Gi

    jobservice:
      existingClaim: "harbor"
      storageClass: ""
      subPath: "jobservice"
      accessMode: ReadWriteOnce
      size: 2Gi

    database:
      existingClaim: "harbor"
      storageClass: ""
      subPath: "db"
      accessMode: ReadWriteOnce
      size: 2Gi

    redis:
      existingClaim: "harbor"
      storageClass: ""
      subPath: "redis"
      accessMode: ReadWriteOnce
      size: 1Gi

  imageChartStorage:
    disableredirect: false

    type: filesystem

    filesystem:
      rootdirectory: /storage

imagePullPolicy: IfNotPresent

logLevel: debug

harborAdminPassword: "harbor_secret"
secretKey: "not-a-secure-key"

notary:
  enabled: false

database:
  type: internal
  internal:
    image:
      repository: goharbor/harbor-db
      tag: dev
    password: "changeit"
    nodeSelector: {}
    tolerations: []
    affinity: {}
  podAnnotations: {}

redis:
  type: internal
  internal:
    image:
      repository: goharbor/redis-photon
      tag: dev
    nodeSelector: {}
    tolerations: []
    affinity: {}
  podAnnotations: {}
```

The new log lines after trying to log in are the following:

```
{"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/login\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/roundrobin/rr: begin ServeHttp on request","time":"2019-04-27T18:05:33Z"}
{"ForwardURL":{"Scheme":"http","Opaque":"","User":null,"Host":"10.244.2.51:8080","Path":"","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":""},"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/login\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/roundrobin/rr: Forwarding this request to URL","time":"2019-04-27T18:05:33Z"}
{"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"10.244.2.51:8080\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/forward: begin ServeHttp on request","time":"2019-04-27T18:05:33Z"}
{"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"10.244.2.51:8080\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/forward/http: begin ServeHttp on request","time":"2019-04-27T18:05:33Z"}
{"level":"debug","msg":"vulcand/oxy/forward/http: Round trip: http://10.244.2.51:8080, code: 404, Length: 1090, duration: 18.860491ms tls:version: 303, tls:resume:true, tls:csuite:c02f, tls:server:hub.nicklehmann.me","time":"2019-04-27T18:05:33Z"}
{"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"10.244.2.51:8080\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/forward/http: completed ServeHttp on request","time":"2019-04-27T18:05:33Z"}
{"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"10.244.2.51:8080\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/forward: completed ServeHttp on request","time":"2019-04-27T18:05:33Z"}
{"Request":"{\"Method\":\"POST\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/login\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"application/json, text/plain, */*\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"de-DE,en-US;q=0.7,en;q=0.3\"],\"Content-Length\":[\"38\"],\"Content-Type\":[\"application/x-www-form-urlencoded\"],\"Cookie\":[\"harbor-lang=en-us; sid=2d2bf5b63fff06e946a4ff8cd0a44a27\"],\"Dnt\":[\"1\"],\"Referer\":[\"https://hub.nicklehmann.me/harbor/sign-in\"],\"Te\":[\"trailers\"],\"User-Agent\":[\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0\"],\"X-Forwarded-Prefix\":[\"/c/\"]},\"ContentLength\":38,\"TransferEncoding\":null,\"Host\":\"hub.nicklehmann.me\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.244.0.1:63545\",\"RequestURI\":\"/login\",\"TLS\":null}","level":"debug","msg":"vulcand/oxy/roundrobin/rr: completed ServeHttp on request","time":"2019-04-27T18:05:33Z"}
```

But the logs in the browser were the most insightful. It seems like neither the `systeminfo` API endpoint neither the one for login is reachable.

![Screenshot 2019-04-27 at 20 02 07](https://user-images.githubusercontent.com/2708875/56854839-5e729580-693d-11e9-8408-af25b49728cc.png)
![Screenshot 2019-04-27 at 20 02 27](https://user-images.githubusercontent.com/2708875/56854842-63374980-693d-11e9-8052-d1a29d073527.png)

And I discovered the following errors in the console:

![Console](https://user-images.githubusercontent.com/2708875/56854845-721dfc00-693d-11e9-946e-4b8ce1c3a4ba.png)

Hope this helps and the issue can be fixed soon!

**Comment by forbearing** [comment]
my expose type is nodePort.  modify the values.yaml file, "externalURL: https" change to "externalURL: http"
 
before: externalURL: https://10.240.11.10:30002
after:    externalURL: http://10.240.11.10:30002

**Comment by QiuToo** [comment]
我也遇到了这个问题，Pod状态正常，使用
![image](https://user-images.githubusercontent.com/88759374/162664085-b8956b96-1908-488c-b6eb-c147832b0875.png)
![image](https://user-images.githubusercontent.com/88759374/162664106-5a167c89-59c2-48e8-9b20-b46f938a4900.png)
![image](https://user-images.githubusercontent.com/88759374/162664140-38c1e55d-8a41-4252-9ebf-f2b5a2098437.png)
希望能获得帮助，非常感谢！
